spec/factories/comments.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :comment do
```

Comment:
Let factory bot know about your model associations, like so:

```
FactoryBot.define do
  factory :comment do
    message { "Lorem Ipsum Text" }

    association :post 
    association :user
  end
end
```

This will allow FactoryBot to create a Post and User every time a Comment is created.